index.html

```diff
@@ -0,0 +1,183 @@
+            #subscribed{
+                float: left;
+                position: relative;
+                top : -8px;
+            }
+            span {
+                color : #646464;
+                font-weight: normal;
+                font-size: 12px;
+            }
+            #registration {
+                text-transform: uppercase;
+                color : white;
+                background-color: #CC3532;
+                width : 120px;
+                float:right;
+                text-align: center;
+                vertical-align: middle;
+                padding-top: 7px;
+                padding-bottom: 7px;
+            }
+            #suggestions {
+                float:left;
+                width:350px;
+            }
+            .all{
+                background-color: black;
+                color: white;
+                width: 60px;
+                text-align: center;
+                border-radius: 16px;
+                height : 25px;
+                padding-top:5px;
+                margin-right: 20px;
+            }
+            .seen{
+                background-color:#D6D6D6;
+                color: black;
+                width: 100px;
+                text-align: center;
+                border-radius: 16px;
+                height : 25px;
+                padding-top:5px;
+            }
+            #selection{
+                display: flex;
+                direction: row;
+                
+            }
+        </style>
+
+    </head>
+    <nav>
+        <a id="icone" href="#"><img src="static/images/icone_image.png"></a>
+        <a id="sigle" href="#"><img src="static/images/youtube_icone.png"></a>
+        <input type="search" placeholder="Rechercher" id="sch">
+        <img id="text_sch"src="static/images/search_icone.png"/>
+        <img id="voice_sch" src="static/images/sound_search.png"/>
+        <img id="video_sch" src="static/images/add_video.png"/>
+        <img id="neuf_carres" src="static/images/neuf_carres.png"/>
+        <img id="notification" src="static/images/notification.png"/>
+        <img id="acompte_sigle" src="static/images/compte_sigle.png"/>
+    </nav>
+    <section >
+        <iframe width="100%" height="500" src="https://www.youtube.com/embed/V74l_zS1x8E" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+    </section>
+    <section id="suggestion_comments">
+      <div id="comments">
+        <h2>Intégrer des vidéos et des playlists</h2>
+        <p id="visibility"><img src="static/images/joint.png" id="joint_image"> Non répertoriée</p>
+        <p id="views">315314 vues <sup> . </sup> 13 mai 2016 </p>
+        <hr>
+        <p><img id="yot" src="static/images/youtube_.jpg"/> 
+            <p id="subscribed">Assistance YouTube <img src="static/images/valide_icone.png"/><br>
+             <span>8,63 M d'abonnés</span>
```

Comment:
You can wrap this part inside ul tag and each element inside the created ul. This could give more semantic sens to your application.